api/src/modules/sourcing-locations/sourcing-location.entity.ts

```diff
@@ -119,14 +134,21 @@ export class SourcingLocation extends TimestampedBaseEntity {
 
   @ManyToOne(() => BusinessUnit, (bu: BusinessUnit) => bu.sourcingLocations, {
     eager: false,
+    onDelete: 'CASCADE',
```

Comment:
I wouldnt cascade delete here.

Reply:
@tiagojsag this is not my whim, after having a chat with Elena it was clear that a SourcingLocations needs a Business-Unit, always. Is there any reason to keep this? We could set a default value for businessUnitId in case that Business Unit gets deleted, but I think we'd be going against business-logic. This also could affect the 10000000 filters FE is implementing

Reply:
so I understood that what cascade meant here was that if you deleted the sourcing location, the business unit would be deleted too. Stackoverflow says otherwise (which makes more sense anyways) so if that's the case, you are right, and it should be left as it is

Reply:
@tiagojsag the other way around. Same for Sourcing-Records and Sourcing-Locations. If a Sourcing-Location gets deleted, all its 'dependant' Sourcing-Records will be deleted to